File: language/benchmarks/tokens/cases/world-log-test/world-log-test.ts
import { println } from "./io";
import { contains } from "./test-log";
import { capture } from "./world-log";

export function main():void{}

export function testWorldsCaptureLogs():boolean{
  const log=capture();
  println(log,"temporary");
  return contains(log,"temporary");
}

export function testWorldsStartFresh():boolean{
  const log=capture();
  return !contains(log,"temporary");
}

File: language/benchmarks/tokens/cases/world-log-test/world-log-test.py
from io_world import println
from test_log import contains
from world_log import capture


def main() -> None:
    pass


def test_worlds_capture_logs() -> bool:
    log = capture()
    println(log, "temporary")
    return contains(log, "temporary")


def test_worlds_start_fresh() -> bool:
    log = capture()
    return not contains(log, "temporary")
